Examples/MAX32650/OTP_Dump/README.md

```diff
@@ -0,0 +1,44 @@
+## Description
+
+This example demonstrates read/write to OTP area inside device's internal memory. Write test irreversibly modifies the memory and it is disabled by default.
+
+## Required Connections
+
+### MAX32650 Evaluation Kit
+-   Connect a USB cable between the PC and the CN2 (USB TO UART) connector.
+-   Connect a debugger to J3 or J4 (JTAG/SWD).
+-   Select RX SEL and TX SEL on headers JP12 and JP13.
+-   Open a terminal application on the PC and connect to the EV kit's console UART at 115200, 8-N-1.
+
+### MAX32650FTHR Evaluation Kit
+-   Connect a USB cable between the PC and the J1 connector.
+-   Connect a debugger to J5.
+-   Open a terminal application on the PC and connect to the EV kit's console UART at 115200, 8-N-1.
+
+## Expected Output
+
+```
+***** OTP Memory Read/Write Example *****
+***** This example demonstrates how you can read/write OTP memory *****
+
+
+***** MAXIM AREA *****
```

Comment:
Ditto

"Update to "ADI AREA" instead? If applicable, we should probably start replacing the "Maxim" names with "ADI" or something similar."